### DISCUSSION TITLE
GitHub newsletter foreground/background color contrast

### DISCUSSION THREAD

**Discussion by pamelafox** [community/community#54307] [Accessibility] [answered]
### Select Topic Area

Product Feedback

### Body

I would prefer higher contrast on the GitHub Insiders newsletter. I think the current contrast might technically pass the WCAG standards, but it is pretty close to failing, and I find it hard to read. Foreground is #8B949E, background is black (I assume).

Thanks!

<img width="759" alt="Screenshot of WCAG contrast checker" src="https://user-images.githubusercontent.com/297042/235808573-fecab6b3-b8f6-4094-8646-021f260bf2bc.png">

<img width="942" alt="Screenshot of GitHub newsletter" src="https://user-images.githubusercontent.com/297042/235808576-5f4394c1-77a0-4f10-a9c0-4e858e093e38.png">

**Comment by cehfisher** [comment]
Hi @pamelafox - I wanted to follow up on this feedback and ask if it still an issue for you? If it is not, please close out this ticket. If it is still an issue, please let us know and we can follow-up with the marketing team. Thank you! (cc @queenofcorgis)

**Comment by pamelafox** [comment, marked as answer]
The newsletters are now arriving formatted with black text on white background (at least the one I found from 1/11/2024 - The GitHub Insider Newsletter), so this is not a problem currently. Thank you!
